doc/contributing/advocacy-ambassador-program.md

```diff
@@ -218,3 +218,31 @@ Some of the things to highlight include:
 * @marco-ippolito
 
 You can find their contact email in the [`README.md`](../../README.md#tsc-technical-steering-committee)
+
+#### Node.js Type Stripping
+
+##### Goal
+
+The goal is to raise awareness of the Node.js TypeScript Type Stripping in the JavaScript ecosystem.
+Some of the things to highlight include:
+
+* The new feature and its benefits and limitations.
```

Comment:
nit: I find it weird to ask to highlight the whole feature, it's kind of already covered by the previous sentence
```suggestion
* The benefits and limitations of the current implementation.
```